### ISSUE TITLE
v-on-function-call with dots in name

### ISSUE THREAD

**Issue by rightaway** [vuejs/eslint-plugin-vue#1085]
**Tell us about your environment**	
* **ESLint version:** 6.8.0
* **eslint-plugin-vue version:**	6.2.2
* **Node version:**	13.9.0

**Please show your full configuration:**

```json
"vue/v-on-function-call": ["error", "never"]
```

**What did you do?**	

```vue
<MyComponent @myevent="$refs.myref.mymethod()"/>
```

**What did you expect to happen?**
Should be no error because you're not allowed to call it without `()` when calling it this way. Even with `never` as the argument you must include a dot. The only time you're allowed to skip the dot is when calling a function like `@myevent="mymethod"`, like when it's in the `methods` of your Vue component.

**What actually happened?**

`Method calls without arguments inside of 'v-on' directives must not have parentheses.eslint(vue/v-on-function-call)`

**Comment by ota-meshi** [comment]
Thanks for this problem.

I think it's a bug.

**Comment by rightaway** [comment]
Are there any cases where it is valid in Vue to have dots in the name but not have `()`, like if calling a method on `$parent`? These cases could be documented in this rule so that people know to add an `eslint-disable` for those cases when using this rule.
